pulsar-broker/src/test/java/org/apache/pulsar/broker/web/ProcessHandlerFilterTest.java

```diff
@@ -0,0 +1,86 @@
+        doReturn(mockConfig).when(mockPulsarService).getConfig();
+        doReturn(Sets.newHashSet("Interceptor1", "Interceptor2")).when(mockConfig).getBrokerInterceptors();
+        ProcessHandlerFilter processHandlerFilter = new ProcessHandlerFilter(mockPulsarService);
+        processHandlerFilter.doFilter(mockHttpServletRequest, mockHttpServletResponse, mockFilterChain);
+        verify(spyInterceptor).onFilter(mockHttpServletRequest, mockHttpServletResponse, mockFilterChain);
+    }
+
+    @Test
+    public void testChainDoFilter() throws ServletException, IOException {
+        PulsarService mockPulsarService = mock(PulsarService.class);
+        BrokerInterceptor spyInterceptor = mock(BrokerInterceptor.class);
+        HttpServletResponse mockHttpServletResponse = mock(HttpServletResponse.class);
+        ServiceConfiguration mockConfig = mock(ServiceConfiguration.class);
+        FilterChain spyFilterChain = spy(FilterChain.class);
+        doReturn(spyInterceptor).when(mockPulsarService).getBrokerInterceptor();
+        doReturn(mockConfig).when(mockPulsarService).getConfig();
+        doReturn(Sets.newHashSet()).when(mockConfig).getBrokerInterceptors();
+        // empty interceptor list
+        HttpServletRequest mockHttpServletRequest = mock(HttpServletRequest.class);
+        ProcessHandlerFilter processHandlerFilter = new ProcessHandlerFilter(mockPulsarService);
+        processHandlerFilter.doFilter(mockHttpServletRequest, mockHttpServletResponse, spyFilterChain);
+        verify(spyFilterChain).doFilter(mockHttpServletRequest, mockHttpServletResponse);
+        clearInvocations(spyFilterChain);
+        // request has MULTIPART_FORM_DATA content-type
+        doReturn(Sets.newHashSet("Interceptor1","Interceptor2")).when(mockConfig).getBrokerInterceptors();
+        HttpServletRequest mockHttpServletRequest2 = mock(HttpServletRequest.class);
+        doReturn(MediaType.MULTIPART_FORM_DATA).when(mockHttpServletRequest2).getContentType();
+        ProcessHandlerFilter processHandlerFilter2 = new ProcessHandlerFilter(mockPulsarService);
+        processHandlerFilter2.doFilter(mockHttpServletRequest2, mockHttpServletResponse, spyFilterChain);
+        verify(spyFilterChain).doFilter(mockHttpServletRequest2, mockHttpServletResponse);
+        clearInvocations(spyFilterChain);
+        // request has APPLICATION_OCTET_STREAM content-type
+        doReturn(Sets.newHashSet("Interceptor1","Interceptor2")).when(mockConfig).getBrokerInterceptors();
+        HttpServletRequest mockHttpServletRequest3 = mock(HttpServletRequest.class);
+        doReturn(MediaType.APPLICATION_OCTET_STREAM).when(mockHttpServletRequest3).getContentType();
+        ProcessHandlerFilter processHandlerFilter3 = new ProcessHandlerFilter(mockPulsarService);
+        processHandlerFilter3.doFilter(mockHttpServletRequest3, mockHttpServletResponse, spyFilterChain);
+        verify(spyFilterChain).doFilter(mockHttpServletRequest3, mockHttpServletResponse);
```

Comment:
Could you please refine these lines? Looks only the MediaType different here.

Reply:
OK, I will refine these.